src/reset.css

```diff
@@ -0,0 +1,13 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap");
+
+.reset,
+.page {
```

Comment:
Why don't you use "reset" class to reset all that you need? I don't think that's a big mistake but it would be much better to reset all default values by one class

Reply:
Agree